src/components/content-tab-snapshots.tsx

```diff
@@ -68,6 +70,13 @@ function SnapshotRow( {
 	const deleteDemoSiteOfflineMessage = __(
 		'Deleting a demo site requires an internet connection.'
 	);
+	const getLastUpdateTimeText = () => {
+		if ( ! date ) {
+			return __( 'Never updated' );
```

Comment:
I think this Never updated won’t ever be displayed.
No need to change it, it's just an observation.

Reply:
Thank you for your review and feedback, Antonio. Yes it should never display. I left it there as a fallback in case the date is not available for some reason. But now thinking about it further, maybe instead we should hide the tooltip in that case completely.